onnxruntime/test/contrib_ops/packed_attention_op_test.cc

```diff
@@ -481,7 +485,21 @@ TEST(PackedAttentionTest, test_on_random_data) {
   }
 }
 
-TEST(PackedAttentionTest, test_on_random_data_large_seq) {
+TEST(PackedAttentionTest, TestWithRandomDataWithRBP) {
+  std::string onnx_model_fp16 = "testdata/packed_attention_fp16.rbp.onnx";  // mainly for cutlass
+  for (int batch_size : std::vector<int>({1, 2, 3, 4, 5, 6, 7, 8})) {
+    for (int sequence_length : std::vector<int>({32, 48, 64, 95, 128})) {
```

Comment:
I'm curious, do we need the `std::vector<int>(...)` around the initializer_list? Or can it just be:

```
for (int batch_size : {1, 2, 3, 4, 5, 6, 7, 8}) {
    for (int sequence_length : {32, 48, 64, 95, 128})
```